docs/neuroglancer_precomputed.md

```diff
@@ -28,7 +28,7 @@ WEBKNOSSOS expects the following file structure for Neuroglancer Precomputed dat
 ```
 my_dataset.precomputed             # One root folder per dataset
 ├─ info                            # Dataset [metadata in JSON format](https://github.com/google/neuroglancer/blob/master/src/neuroglancer/datasource/precomputed/volume.md#info-json-file-specification)
-├─ scale_1                         # One subdirectory with the same name as each scale/magnification "key" value specified in the info file. Each subdirectory contains a chunked representation of the data for a single resolution.
```

Comment:
@fm3 I see your point that "resolution" made sense in the other places, so I reverted the changes. But here, I am not sure I understand. I think "resolution" can be used as a synonym for "mag" to explain the concept, and from there on, only "mag(nification)" should be used. The latter seems to be the case here. What am I missing?

Reply:
that this is specifically about the “neuroglancer precomputed” file format, so it makes sense to follow their terminology. In fact, it looks like this sentence is a [direct quote from here](https://github.com/google/neuroglancer/blob/master/src/datasource/precomputed/volume.md). However, maybe both terms can be used to bridge the gap to webknossos. How about something like “contains a chunked representation of the data for a single resolution (meaning single magnification/mag for WEBKNOSSOS)”?

Reply:
Oh, I see. Yes, good idea!